### PR TITLE
Adds Rabbit Lop-Eared to ears list

### PR DESCRIPTION
**Pull request by Kyutness**
## About The Pull Request
Adds another 'ear' option, one that I in particular really like the look of, and if others do too, then even better!

## Why It's Good For The Game
One more cosmetic option.

## Changelog
:cl:
add: Rabbit Lop-Eared
/:cl:

### CODE REVIEW

File: code/modules/mob/dead/new_player/sprite_accessories/ears.dm [hunk 1]
```diff
@@ -171,6 +171,12 @@
 	color_src = MATRIXED
 	icon = 'modular_citadel/icons/mob/mam_ears.dmi'
 
+/datum/sprite_accessory/ears/human/rabbit
+	name = "Rabbit Lop-Eared"
+	icon_state = "rabbitlop"
+	color_src = MATRIXED
+	icon = 'modular_citadel/icons/mob/mam_ears.dmi'
```
Comment (jjpark-kb) [inline code/modules/mob/dead/new_player/sprite_accessories/ears.dm:178]: make it modular

File: code/modules/mob/dead/new_player/sprite_accessories/ears.dm [hunk 2]
```diff
@@ -323,3 +329,7 @@
 /datum/sprite_accessory/mam_ears/bunny
 	name = "Bunny"
 	icon_state = "bunny"
+
+/datum/sprite_accessory/mam_ears/Rabbit_Lop
+	name = "Rabbit Lop-Eared"
+	icon_state = "rabbitlop"
```
Comment (jjpark-kb) [inline code/modules/mob/dead/new_player/sprite_accessories/ears.dm:335]: make it modular

**Comment by jjpark-kb** [comment]
you added icons to modular_citadel dmis
you cannot do that. You must create icons in modular_skyrat

File: code/modules/mob/dead/new_player/sprite_accessories/ears.dm (hunk 1 above)
Comment (Kyutness) [reply code/modules/mob/dead/new_player/sprite_accessories/ears.dm]: Honestly not exactly sure what that means, but I tried to do it anyways.

**Pull request merged**